gnome-extra/gnome-commander/metadata.xml

```diff
@@ -1,21 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <!--maintainer-needed-->
-  <upstream>
-    <maintainer status="active">
-      <email>u.scholz83@gmx.de</email>
-      <name>Uwe Scholz</name>
-    </maintainer>
-    <changelog>https://gitlab.gnome.org/GNOME/gnome-commander/blob/gcmd-1-4/NEWS</changelog>
-    <bugs-to>https://gitlab.gnome.org/GNOME/gnome-commander/issues</bugs-to>
-    <remote-id type="freecode">gnome-commander</remote-id>
-  </upstream>
-  <use>
-    <flag name="chm">Enable metadata support for Microsoft Compiled HTML Help files</flag>
-    <flag name="gsf">Add metadata support for OLE, OLE2 and ODF</flag>
-    <flag name="unique">Enable single instance support using dev-libs/libunique</flag>
-    <flag name="pdf">Add metadata support for PDF</flag>
-    <flag name="taglib">Add metadata support for ID3, Vorbis, FLAC and APE</flag>
-  </use>
+  <maintainer type="person">
+    <email>u.scholz83@gmx.de</email>
+    <name>Uwe Scholz</name>
+  </maintainer>
+<maintainer type="project">
+    <email>proxy-maint@gentoo.org</email>
+    <name>Proxy Maintainers</name>
+  </maintainer>
```

Comment:
Please just add yourself+proxy-maint to metadata.xml, in line with existing indentation. It should not lead to all these unrelated line changes.

Reply:
Well, I don't know how this happened. I was just doing a "git revert a4ca3f308"... Trying to fix it.